zc_common/fields.py

```diff
@@ -9,6 +10,10 @@
 US_PHONE_FORMAT = r'^(\+1\s?)?\(?(\d{3})\)?[\s-]?(\d{3})[\s-]?(\d{4})$'
 
 
+def numeric_uuid_generator():
+    return str(uuid.uuid4().int)[:10]
```

Comment:
Why? If we're going to default to a uuid-based key, at least use the full uuid4.